views/windows.tmpl

```diff
@@ -109,15 +109,14 @@
                       <th class="text-center d-none d-sm-table-cell">Difficulty</th>
                       <th class="text-center"><span class="d-none d-sm-inline">Ticket </span>Price<span class="d-none d-sm-inline"> (DCR)</span></th>
                       <th class="text-right pr-0 jsonly">Age</th>
-                      <th class="text-right pr-0 d-none d-sm-table-cell">Start Time</th>
+                      <th class="text-right pr-0 d-none d-sm-table-cell">Start Time (UTC)</th>
                   </tr>
               </thead>
               <tbody>
               {{range .Data}}
                   <tr>
                       <td class="text-left pl-0">
-                          {{$blocksCount := (.BlocksCount)}}
-                          {{if lt .BlocksCount $.WindowSize}}{{$blocksCount = (subtract .BlocksCount 1)}} <span>({{$blocksCount}}/{{$.WindowSize}}) </span>{{end}}
+                          {{if lt .BlocksCount $.WindowSize}} <span>({{.BlocksCount}}/{{$.WindowSize}}) </span>{{end}}
```

Comment:
For these kinds of changes, screenshots of the edge cases need to be shown. First and last block in a window.

Reply:
Screenshot added.